src/vulkan/resource.cc

```diff
@@ -191,8 +191,8 @@ Resource::AllocateResult Resource::AllocateAndBindMemoryToVkImage(
     VkDeviceMemory* memory,
     VkMemoryPropertyFlags flags,
     bool force_flags) {
-  if (image == nullptr)
-    return {Result("Vulkan::Given VkImage pointer is nullptr"), 0};
+  if (image == VK_NULL_HANDLE)
```

Comment:
This can move out to a new change and land pretty quickly.

Reply:
done